### ISSUE TITLE
excavator-benchmark hasn't been updated to include neoscrypt and nist5

### ISSUE THREAD

**Issue by gordan-bobic** [nicehash/excavator#117]
As per the title.

**Comment by kyleboddy** [comment]
NIST5 would be a big addition for sure.

**Comment by gordan-bobic** [comment]
Here is a patch for excavator-benchmark that rectifies this:

```
--- excavator-benchmark.orig	2018-01-16 00:49:59.078798339 +0000
+++ excavator-benchmark	2018-01-16 00:50:33.067018635 +0000
@@ -79,7 +79,7 @@
 
 echo Benchmarking with $(( device_end-device_start+1 )) device"("s")" for ${length} seconds for each algorithm...
 echo Benchmarking without network connection first...
-for algo in equihash pascal decred sia lbry blake2s lyra2rev2 cryptonight daggerhashimoto
+for algo in nist5 neoscrypt equihash pascal decred sia lbry blake2s lyra2rev2 cryptonight daggerhashimoto
 do
     config_file=$(mktemp)
     output_file=$(mktemp)
@@ -147,7 +147,7 @@
 
 echo Benchmarking with $(( device_end-device_start+1 )) device"("s")" for ${length} seconds for each algorithm...
 echo Connectng to ${server}...
-for algo in equihash pascal decred sia lbry blake2s lyra2rev2 cryptonight daggerhashimoto
+for algo in nist5 neoscrypt equihash pascal decred sia lbry blake2s lyra2rev2 cryptonight daggerhashimoto
 do
     case $algo in
     equihash*)
```

**Comment by dropky** [comment]
This should be fixed with v1.4.1.
